MonoGame.Framework/Content/ContentTypeReaderManager.cs

```diff
@@ -35,40 +35,57 @@ namespace Microsoft.Xna.Framework.Content
 {
     public sealed class ContentTypeReaderManager
     {
-        private static readonly object _locker;
-
+        private static readonly object _locker;
+
         private static readonly Dictionary<Type, ContentTypeReader> _contentReadersCache;
 
-        private Dictionary<Type, ContentTypeReader> _contentReaders;
+        private Dictionary<Type, ContentTypeReader> _contentReaders;
 
-		private static readonly string _assemblyName;
-		
+        private static readonly string _assemblyName;
 
-		static ContentTypeReaderManager()
-		{
-            _locker = new object();
+        /// <summary>
+        /// Regex to parse a Generic Type string.
+        /// Capture Groups:
+        /// 0 - Full String
+        /// 1 - Name up to `
+        /// 2 - Number of Generic Parameters
+        /// 3 - [
+        /// 4 - Collection of Generic types
+        /// 5 - ]
+        /// 6 - Optional ', AssemblyName'
+        /// 
+        /// Windows 8 RT doesn't appear to support Compiled Regex.
+        /// </summary>
+#if (WINDOWS_STOREAPP || WINDOWS_PHONE)
```

Comment:
I initially believed so, but after recalling that the Win10 build passed, I took another look. UAP appears to have the Compiled support that WIn8 did not.